cdc/sinkv2/eventsink/event_appender.go

```diff
@@ -69,26 +69,16 @@ func (t *TxnEventAppender) Append(
 		}
 
 		lastTxn := buffer[len(buffer)-1]
-		// Normally, this means the commitTs grows.
-		if lastTxn.GetCommitTs() != row.CommitTs ||
-			// Normally, this means we meet a new big txn batch.
-			row.SplitTxn ||
-			// Normally, this means we meet a new txn.
-			lastTxn.StartTs < row.StartTs {
-			// Fail-fast check
-			commitTsDecreased := lastTxn.GetCommitTs() > row.CommitTs
-			if commitTsDecreased {
-				log.Panic("The commitTs of the emit row is less than the received row",
-					zap.Uint64("lastReceivedCommitTs", buffer[len(buffer)-1].GetCommitTs()),
-					zap.Any("row", row))
-			}
-			startTsDecreased := lastTxn.StartTs > row.StartTs
-			if startTsDecreased {
-				log.Panic("The startTs of the emit row is less than the received row",
-					zap.Any("lastReceivedStartTs", buffer[len(buffer)-1].GetCommitTs()),
-					zap.Any("row", row))
-			}
 
+		lastCommitTs := lastTxn.GetCommitTs()
```

Comment:
Why do we remove other checks?

Reply:
startTs can fallback, it's expected. So I removed it.